lms/djangoapps/certificates/management/commands/cert_whitelist.py

```diff
@@ -69,19 +81,13 @@ def handle(self, *args, **options):
 
         if options['add'] or options['del']:
             user_str = options['add'] or options['del']
-            if '@' in user_str:
-                user = User.objects.get(email=user_str)
+            add_to_whitelist = True if options['add'] else False
+            if "," in user_str:
```

Comment:
You can split always here and treat the output as a list.  Doing so, you can remove much of the code here.